fiftyone/server/routes/temporal_tags.py

```
@@ -0,0 +1,462 @@
+
+    return {
+        "tags": [
+            _serialize_temporal_tag(tag)
+            for tag in _handle_temporal_tag_errors(
+                lambda: fota.list_temporal_tags(dataset, filter=tag_filter)
+            )
+        ]
+    }
+
+
+def _get_dataset_from_request(request: Request):
```

Ruff (0.16.0)

[warning] Missing return type annotation for private function `_get_dataset_from_request`

(ANN202)